docs/api/visualizer/create.md

```diff
@@ -0,0 +1,178 @@
+---
+id: create
+title: Create a new Visualizer
+sidebar_label: Create
+keywords:
+  - OpenHIM
+  - API
+  - Visualizer
+  - Create
+description: Create an OpenHIM Visualizer via the API
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Create visualizer
+
+To create a new visualizer record you will need to make a TLS request to the OpenHIM API for the below method and endpoint and supply the JSON object for the visualizer record.
+
+
+```curl
+Method: POST
+Endpoint: /visualizers
```

Comment:
Perhaps to be a bit more verbose for new users
```suggestion
Endpoint: {openhim_url}:8080/visualizers
```

Reply:
Great thanks @MattyJ007 good suggestion

Reply:
done in 5a34e40